src/lib.rs

```diff
@@ -1737,6 +1708,12 @@ mod test {
                         conn_count: 4,
                         skip_blocks: 0,
                     },
+                    #[cfg(feature = "rpc")]
```

Comment:
Since this whole tests has `#[cfg(feature = "rpc")]` I don't think you need it here. 
```suggestion
```